acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/cluster/Cluster.java

```diff
@@ -87,12 +87,19 @@ public void start(final List<? extends RunnableNode> nodes) {
 
     nodes
         .parallelStream()
-        .filter(node -> bootnode.map(boot -> boot != node).orElse(true))
+        .filter(
+            node -> {
+              LOG.info("starting non-bootnode {}", node.getName());
+              return bootnode.map(boot -> boot != node).orElse(true);
+            })
         .forEach(this::startNode);
 
     if (clusterConfiguration.isAwaitPeerDiscovery()) {
       for (final RunnableNode node : nodes) {
-        LOG.info("Awaiting peer discovery for node {}", node.getName());
+        LOG.info(
+            "Awaiting peer discovery for node {}, expecting {} peers",
+            node.getName(),
+            nodes.size() - 1);
```

Comment:
can we put a try-catch block around the awaitPeerDiscovery and if fail - dump the actual peers
(this would require creating a transaction in the dsl which wraps Ethereum.adminPeers - similar to NetPeerCountTransaction)

Reply:
because it's an `assert` I think it would involve some pretty major refactoring

Reply:
:( That's the killer piece of info in some respects - but all good - getting this in is worth it.